docs/enums/tableau.analyticsobjecttype.html

```diff
@@ -87,30 +87,30 @@ <h2>Enumeration members</h2>
 				<section class="tsd-panel tsd-member tsd-kind-enum-member tsd-parent-kind-enum">
 					<a name="cluster" class="tsd-anchor"></a>
 					<h3>Cluster</h3>
-					<div class="tsd-signature tsd-kind-icon">Cluster<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-symbol"> =&nbsp;&quot;cluster&quot;</span></div>
+					<div class="tsd-signature tsd-kind-icon">Cluster<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-symbol"> = &quot;cluster&quot;</span></div>
 					<aside class="tsd-sources">
 						<ul>
-							<li>Defined in Namespaces/Tableau.ts:26</li>
+							<li>Defined in src/ExternalContract/Namespaces/Tableau.ts:26</li>
```

Comment:
Do we know why the relative paths here were added?

Reply:
TypeDoc now adds the path from the root in the "Defined in" statement.

Reply:
Great. Thanks for looking that up.
I think we just need to decide what we want this to look like and when. Let's do that in the slack thread then update here.